hydropandas/io/bro.py

```diff
@@ -473,8 +473,18 @@ def get_metadata_from_gmw(bro_id, tube_nr):
     meta = {"monitoring_well": bro_id, "tube_nr": tube_nr, "source": "BRO"}
 
     # x and y
-    xy = gmw.find("dsgmw:deliveredLocation//gmwcommon:location//gml:pos", ns)
-    meta["x"], meta["y"] = [float(val) for val in xy.text.split()]
+    xy_elem = gmw.find("dsgmw:deliveredLocation//gmwcommon:location//gml:pos", ns)
+    xy = [float(val) for val in xy_elem.text.split()]
+
+    # convert crs
+    srsname = gmw.find("dsgmw:deliveredLocation//gmwcommon:location", ns).attrib[
+        "srsName"
+    ]
+    epsg_gwm = int(srsname.split(":")[-1])
+    transformer = Transformer.from_crs(epsg_gwm, epsg)
```

Comment:
are we sure this is correct or do we need the full CRS string definition like we use in nlmod?

Reply:
I added the epsg string for transformations to epsg 28992. I did not use this for transformations from epsg 28992.